#279 & #409: add time based item drop rate limiting (#410)
Imaginary-Horizons-Productions/BountyBot: ChangeLog.md

- Fixed crash when using `/create-default rank-roles` twice in a row
- Fixed `/moderation user-report` listing XP awarded for recent bounties as "undefined"
- Fixed Dark Purple Profile Colorizer not dropping

> - Item drops are now limited to 2 items per day, or 4 items if you are a premium user

Reviewer: ```suggestion
- Item drops are now limited to 2 items per day, or 4 items for Premium users
```
The 2nd person tense sticks out a bit among the 3rd person passive technical documentation is usually written in (also, we should be confident that the feature offerings are indeed "Premium").

Author: I don't know how you're reading a lack of confidence in the old wording, but I like the new wording better anyways.
